install/fossinit.php

```diff
@@ -346,7 +351,7 @@ function explainUsage()
 {
   require_once("$LIBEXECDIR/dbmigrate_multiple_copyright_decisions.php");
 
-  $sysconfig['Release'] = "3.1.0";
+  $sysconfig['Release'] = "3.3.0";
```

Comment:
Doesn't that cause problems with current installations, which have `3.1.0` written in the DB as current datamodel version?

I would find better to not change this line but instead have the following additional lines:
```php
// fix release-version datamodel-version missmatch
if($isUpdating && (empty($sysconfig['Release']) || $sysconfig['Release'] == '3.1.0')) {
    $sysconfig['Release'] = "3.3.0";
}
```